src/pages/admin-ui/scenario.page.ts

```diff
@@ -0,0 +1,39 @@
+import { Locator, Page } from '@playwright/test';
+import { BasePage, ContentManagementPage } from '.';
+
+export class ScenarioPage extends ContentManagementPage {
+  readonly page: Page;
+  readonly scenarioName: string;
+  readonly newScenarioButton: Locator;
+  readonly saveButtonLink: Locator;
+  readonly cancelButtonLink: Locator;
+  readonly scenarioLink: Locator;
+  readonly newScenarioName: Locator;
+  readonly newScenarioDescription: Locator;
+  readonly newScenarioKeepaliveDuration: Locator;
+  readonly newScenarioPauseDuration: Locator;
+
+  constructor(page: Page, username: string) {
+    super(page, username);
+    this.page = page;
+    this.scenarioLink = page.getByRole('link', { name: 'Scenarios' });
+    this.newScenarioButton = page.getByRole('button', { name: 'New Scenario' });
+    this.newScenarioName = page.getByRole('textbox', { name: 'Name' });
+    this.newScenarioDescription = page.getByRole('textbox', { name: 'Description' });
+    this.newScenarioKeepaliveDuration = page.getByLabel('Keepalive Duration');
+    this.newScenarioPauseDuration = page.getByLabel('Pause Duration');
+    this.saveButtonLink = page.getByRole('button', { name: 'Save' });
+    this.cancelButtonLink = page.getByRole('button', { name: 'Cancel' });
+  }
+
+  async openNewScenarioModalAndSave(scenarioName: string, description: string, keepaliveduration: string, pauseduration: string): Promise<ContentManagementPage> {
```

Comment:
Opening the ScenarioPage (scenarioLink.click()) must be done in ContentManagementPage
It would be better to split this method and provide explicit methods like OpenNewScenarioForm, FillNewScenarioForm() and SaveNewScenarioForm() and call them from ContentManagementPage method openNewScenarioModalAndSave